ex_37_v2.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+# Step 1: import random
+import random
+
+# Step 2: create empty string for new password
+new_password = ""
+
+# Step 3: create lists
+special_characters = ["!", "@", "#", "$", "%", "^", "&", "*", "<", ">"]
```

Comment:
This variable, `numbers` and `letters` all qualify as constants because they won't change all all, and are constant.